types/office-js/index.d.ts

```diff
@@ -13581,19 +13580,20 @@ declare namespace OfficeExtension {
      */
     interface LoadOption {
         /**
-         * Contains a comma delimited list or an array of parameter/relationship names. Optional.
+         * A comma-delimited string, or array of strings, that specifies the properties/relationships to load.
          */
         select?: string | string[];
         /**
-         * Contains a comma delimited list or an array of relationship names. Optional.
+         * A comma-delimited string, or array of strings, that specifies the relationships to load.
          */
         expand?: string | string[];
         /**
-         * Specifies the maximum number of collection items that can be included in the result. Optional. You can only use this option when you use the object notation option.
+         * Only usable on collection types. Specifies the maximum number of collection items that can be included in the result. 
+         * You can only use this option when you use the object notation option.
```

Comment:
Please remove the line "You can only use this option when you use the object notation option. " from the top and skip properties. I don't think that's right. Where are you seeing that?

Reply:
It's in Word.LoadOption: https://dev.office.com/reference/add-ins/word/loadoption

Reply:
OK. I see where the info came from. That was a mistake. (It actually doesn't make any sense in the context of the load method. I think someone got a warning that applies to the set method mixed in here.) At any rate, please delete it.